hack/lib/start.sh

```diff
@@ -162,6 +162,7 @@ function os::start::internal::patch_master_config() {
 	local sudo=${USE_SUDO:+sudo}
 	cp "${SERVER_CONFIG_DIR}/master/master-config.yaml" "${SERVER_CONFIG_DIR}/master/master-config.orig.yaml"
 	oc ex config patch "${SERVER_CONFIG_DIR}/master/master-config.orig.yaml" --patch="{\"etcdConfig\": {\"address\": \"${API_HOST}:${ETCD_PORT}\"}}" | \
+	oc ex config patch - --patch="{\"pluginConfig\": {\"openshift.io/ImagePolicy\": {\"configuration\": {\"apiVersion\": \"v1\", \"executionRules\": [{\"matchImageAnnotations\": [{\"key\": \"images.openshift.io/deny-execution\", \"value\": \"true\"}], \"name\": \"execution-denied\", \"onResources\": [{\"resource\": \"pods\"}, {\"resource\": \"builds\"}], \"reject\": \"true\", \"skipOnResolutionFailure\": \"true\" }], \"kind\": \"ImagePolicyConfig\" }, \"location\": \"\"}}}" | \
```

Comment:
@deads2k fixes tests in https://github.com/openshift/origin/pull/17473

Reply:
> @deads2k fixes tests in #17473

You can save this commit for that pull.  You want to pick up that pull?

Reply:
@deads2k 
> You can save this commit for that pull. You want to pick up that pull?

Sure, will start working on the pull now